core/sbt-cloudflow/src/sbt-test/sbt-cloudflow/decouple-version/build.sbt

```diff
@@ -2,7 +2,8 @@ lazy val helloWorld =  (project in file("."))
     .enablePlugins(CloudflowApplicationPlugin, CloudflowAkkaPlugin)
     .settings(
       scalaVersion := "2.12.11",
-      cloudflowVersion := "2.0.10",
+      // TODO: re-enable this after the first Cloudflow release with cross-built runtime
```

Comment:
Waiting Monday to merge to raise awareness around this test in "pending" state, cc. @RayRoestenburg